cocos2d/core/CCDirector.js

```diff
@@ -1503,7 +1504,7 @@ cc.Director.PROJECTION_CUSTOM = 3;
  * @constant
  * @type {Number}
  */
-cc.Director.PROJECTION_DEFAULT = cc.Director.PROJECTION_3D;
+cc.Director.PROJECTION_DEFAULT = cc.Director.PROJECTION_2D;
```

Comment:
2D projection permit the new vertexZ to take effect, in this version, vertexZ will be distributed from back to front with a 1/100 step value.

And seems 2D projection can solve black line between tile map tiles